test/Atmos/Parameterizations/Microphysics/runtests.jl

```diff
@@ -1,161 +1,11 @@
-using Test
-using ClimateMachine.Microphysics
-using ClimateMachine.Thermodynamics
+using Test, MPI
 
-using CLIMAParameters
-using CLIMAParameters.Planet: ρ_cloud_liq, R_v, grav, R_d, molmass_ratio
-using CLIMAParameters.Atmos.Microphysics: τ_cond_evap, τ_acnv, q_liq_threshold
+include(joinpath(@__DIR__, "..", "..", "..", "testhelpers.jl"))
 
-struct EarthParameterSet <: AbstractEarthParameterSet end
-const param_set = EarthParameterSet()
+@testset "Microphysics tests" begin
 
-@testset "RainDropFallSpeed" begin
-    # two typical rain drop sizes
-    r_small = 0.5 * 1e-3
-    r_big = 3.5 * 1e-3
+    runmpi(joinpath(@__DIR__, "1_unit_tests.jl"))
```

Comment:
Shouldn't we keep these running on slurm rather than having them run both slurm and in unit tests?

Reply:
@charleskawczynski - Sorry. I don't know what is the policy for where to put which tests. If you want to make some changes here please do. I was just trying (and failing) to make them run on Windows on azure. 

In general it would be great if tests `2_saturation_adjustment.jl` and `3_warm_rain.jl` were tested on GPU.

Reply:
Yeah, we should have that documented in the contributing section.

I can help move some things around, and yes, the `2_saturation_adjustment.jl` and `3_warm_rain` will be run on the GPU (along with the slurm tests).